easybuild/easyblocks/n/numpy.py

```diff
@@ -48,6 +48,28 @@
 from distutils.version import LooseVersion
 
 
+def parse_numpy_test_suite_output(full_testsuite_output):
+    # Grab the line of the testcmd_output which contains the summary from the test suite.
+    # Summary looks like:
+    # ========== 4860 passed, 14 skipped, 88 deselected, 7 xfailed, 4 error in 63.72 seconds ==========
+    testsuite_summary = [i for i in full_testsuite_output.split('\n') if
+                         i.startswith('=') and 'passed' in i and 'seconds' in i]
+
+    # Verify we got a summary like expected (should be a list with a single element)
+    if not testsuite_summary:
+        raise EasyBuildError("Could not extract summary from test suite output, is it written over multiple lines?")
+
+    # Use regex to help parse it
+    regexp = r"([\w]+)\ ([\w]+)(?:\ in|,)"
```

Comment:
Are you grabbing the `failed` and `error` values here? If so, this should be improved significantly to actually use `error` and `failed`...

We're only expecting one summary line, correct?

Reply:
I'm grabbing everything in the summary line, the resulting dictionary looks like:
```
{'skipped': 1291, 'warnings': 316969, 'xfailed': 78, 'xpassed': 7, 'failed': 4, 'deselected': 1223, 'passed': 14097}
```

Reply:
> We're only expecting one summary line, correct?

I wasn't sure what happens when the line gets really long, I thought it might split it over two but no, it doesn't. In other words, yes it is just a single summary line.

Reply:
OK, the regex was confusing me a bit, since you're using `\w` twice, both for the count and the 'label'.
Please use `\d` instead to grab the count (this makes the regexp a little bit stricter, but avoids issues with the `int(...)` used below).
Let's clarify the regular expression a bit too, since it's not immediately obvious:

```python
# regex to match all "<count> <label>" instances in test summary
regexp = r"([\d]+)\ ([\w]+)(?:\ in|,)"
```